src/vs/workbench/test/node/api/extHostLanguageFeatures.test.ts

```diff
@@ -567,7 +567,7 @@ suite('ExtHostLanguageFeatures', function () {
 
 		return threadService.sync().then(() => {
 
-			return provideReferences(model, new EditorPosition(1, 2)).then(value => {
+			return provideReferences(model, new EditorPosition(1, 2), (value) => { }).then(value => {
```

Comment:
_Callers_ of `provideReferences` should always pass a progress closure. This is to prevent someone in the VS codebase from forgetting to propagate progress.

_Implementers_ of `provideReferences` can elect to call or not call this closure (and it is ok if they omit this parameter from their function definition).